### DISCUSSION TITLE
Great time to sort out monorepo structure

### DISCUSSION THREAD

**Discussion by timbrinded** [polkadot-api/polkadot-api#64] [Ideas]
Hi all,

very excited to be using the codegen package a go when it's ready. I've been previously using `@polkadot/api-augment` for moonbeam which has a fair amount of friction.

Can i suggest though for Capi that you use industry standard practises for monorepo management? A huge amount of technical debt in polkadot-js because of not using proper workspace manageement facilities. 

Using Yarn or Pnpm's workspace features would have taken care of a lot of this.

I love it how you're using vitest and some modern libs, but srsly - ditch npm 😉

**Comment by ryanleecode** [comment]
Pnpm added!
